app/views/advisories/index.html.erb

```diff
@@ -1,30 +1,45 @@
 <% @meta_description = "Browse all Security Advisories #{params[:severity].try(:humanize)} #{params[:ecosystem]} #{params[:package_name]} #{params[:repository_url]}" %>
 
 <div class="container-sm">
-  <h2 class="h1">
-    <%= @meta_title = "#{params[:severity].try(:humanize)} #{params[:ecosystem]} #{params[:package_name]} Security Advisories"%>
+  <h2 class="h1 mb-5">
+   Browse Security Advisories
   </h2>
 
-  <p class="lead">
-    Browse all Security Advisories
-    <% if params[:severity].present? || params[:ecosystem].present? || params[:package_name].present? || params[:repository_url].present? %>
-      for 
-    <% end %>
-    <%= params[:severity].try(:humanize) %>
-    <%= params[:ecosystem] %>
-    <%= params[:package_name] %>
-    <%= params[:repository_url] %>
-  </p>
+
 
   <div class="row">
     <div class="col-lg-8">
-      <div class="listing-controls text-end">
-        <%= render 'sort' %>
+      <div class="listing-controls p-3 mb-5 rounded">
+
+      <%
+      severity_str = params[:severity].present? ? "<strong>#{params[:severity].humanize}</strong> " : ""
+    
+      details = []
+      details << "<strong>#{params[:package_name]}</strong>" if params[:package_name].present?
+      details << "<strong>#{params[:repository_url]}</strong>" if params[:repository_url].present?
+      package_details = details.any? ? " for #{details.join(', ')}" : ""
+    
+      ecosystem_str = params[:ecosystem].present? ? " in <strong>#{params[:ecosystem]}</strong>" : ""
+    %>
+    
+    <div class="row">
+      <div class="col-8">
+        <p class="mb-0">
+          <%= "#{severity_str}Security Advisories#{package_details}#{ecosystem_str}".html_safe %> 
```

Comment:
this is actually a bit of a security risk, it's allowing any html passed in from the parameters to be rendered in the page, we should move it into a helper method instead